hack/schemapatch-config.yaml

```diff
@@ -9,6 +9,7 @@ k8s.io/api/core/v1.VolumeSource:
     - Secret
     - ConfigMap
     - Projected
+    - EmptyDir
```

Comment:
We should add a `preserveUnknownFields: true # for feature flagged fields` here in accordance with the other types. So far, all of the fields that require a feature flag are not actually in the schema but allowed this way.

Reply:
Ok let me try that, want to check what it means from a validation perspective.